### PR TITLE
Preserve the world camera when switching to/from split view

### PR DESCRIPTION
**Pull request by Garanas**
## Description of the proposed changes

Preserves the settings of the primary world camera when switching to/from split view.

## Testing done on the proposed changes

Start up the game and switch to/from split view.

## Additional context

https://github.com/user-attachments/assets/24cb0417-1aa6-4368-b85c-b48fbdacc453

## Checklist

- [x] Changes are annotated, including comments where useful
- [x] Changes are documented in the changelog for the next game version

### CODE REVIEW

**Comment by Garanas** [comment]
Mhh, some formatting got in too.

**Comment by Garanas** [comment]
@BlackYps Small but nice change to include in the hotfix perhaps.

File: changelog/snippets/features.6576.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+- (#6576) Preserve the world camera when switching to/from split view
+
+When you switch to/from split view the primary camera (the left camera) is now preserved. This makes it less disorientating for a player (or caster) to switch to/from split view.
```
Comment (lL1l1) [inline changelog/snippets/features.6576.md:3]: The indentation from the list is lost when doing a double line break.
```suggestion
  When you switch to/from split view the primary camera (the left camera) is now preserved. This makes it less disorientating for a player (or caster) to switch to/from split view.
```
- example

double line break

```
- example

double line break
```

- example 2
single line break

```
- example 2
single line break
```

**Review by lL1l1 — COMMENTED** [review]
The actual changes work and look good. I considered whether or not the right view's camera settings should preserved, but since it's not as persistent as the left view it doesn't need preservation (although an argument could be made for preference of left/right as the "main" view).

The formatting of long statements into multiline fragments is annoying (for me at least), maybe that could be changed in the repository's default vscode settings file, to more officially/consistently make it not happen? Currently some files have that formatting, while others don't. In particular I think the Prettier formatter doesn't like the blueprint file formatter, and conflicts on the format of sound blueprint tables.

**Comment by Garanas** [comment]
> The formatting of long statements into multiline fragments is annoying (for me at least), maybe that could be changed in the repository's default vscode settings file, to more officially/consistently make it not happen?

I agree, I think it is a small change to let prettier ignore it. I'd need to look into it.

**Review by lL1l1 — APPROVED** [review]

**Comment by lL1l1** [comment]
Prettier doesn't get used for Lua files, the extension's own formatter is, and it has a weird config format: the `settings.json` format is the key-value pair as strings, and the `.editorconfig` file has to be at the root and is in its own format. It also requires the language to be specified before editing the options.

**Pull request merged**